.github/workflows/deploy.yml

```diff
@@ -84,7 +88,7 @@ jobs:
           TF_VAR_dockerhub_password: ${{ secrets.DOCKERHUB_PERSONAL_ACCESS_TOKEN }}
 
       - name: Trigger ${{ env.NEXT_ENV }} Deployment
-        if: ${{ success() && env.NEXT_ENV != '' && github.ref == 'refs/heads/master' }}
+        if: ${{ success() && env.NEXT_ENV != '' && github.ref == 'refs/heads/421-deploy-to-register-app-to-staging' }}
```

Comment:
can you revert this? also in build.yml

Reply:
i actually did; but forgot to push up....lol